### DISCUSSION TITLE
teleport replace 2FA with Radius authentication

### DISCUSSION THREAD

**Discussion by muzammel111** [gravitational/teleport#7491] [Q&A]
Dear Community, i want to integrate/replace teleport 2FA with our custom radius/OTP . Is is possible ? I believe as open source nature it will possible :) . So what are the measures we want to take to achieve this as its difficult to maintain multiple otp's for large number of users. Please guide. 

Thanks.

**Comment by webvictim** [comment]
It might be possible with a PR, but it's not something on Teleport's roadmap currently.

**Comment by muzammel111** [comment]
Thankyou @webvictim for your response. What is mean by PR ? please guide

Thanks.

**Reply by webvictim** [reply]
A pull request - i.e. write the code to add RADIUS support to Teleport and submit it for inclusion in the project.

**Comment by muzammel111** [comment]
Thanks @webvictim for explain. I am sysadmin, not a developer :-), hope some one will work on it soon.
